dask_ml/model_selection/_incremental.py

```diff
@@ -879,10 +879,17 @@ def _get_params(self):
     def _additional_calls(self, info):
         calls = {k: v[-1]["partial_fit_calls"] for k, v in info.items()}
 
+        if self.patience < 0 or self.patience == 1:
```

Comment:
Is patience between 0 and 1 OK?

Reply:
This is protecting against the default case when `patience=False`. I've made that more explicit in my tests and code.